pinot-controller/src/main/java/org/apache/pinot/controller/api/resources/PinotSegmentRestletResource.java

```diff
@@ -635,8 +635,8 @@ public ServerReloadControllerJobStatusResponse getReloadJobStatus(
     Map<String, List<String>> serverToSegments;
 
     String singleSegmentName = null;
-    if (controllerJobZKMetadata.get(CommonConstants.ControllerJob.JOB_TYPE)
-        .equals(ControllerJobType.RELOAD_SEGMENT.toString())) {
+    if (controllerJobZKMetadata.get(CommonConstants.ControllerJob.SEGMENT_RELOAD_JOB_SEGMENT_NAME)
+       != null) {
```

Comment:
curious why change the content of the controller zk metadata to use the segment name for checking whether it is reloading 1 or all segments? what's the pro/con putting the together vs separate? 

just my opinion, i know they are related, but i dont foresee any circumstances we will use the 2 types of reload together (maybe I am wrong please definitely share your thoughts)

Reply:
Yes, reload and reload_all are related, so combined into a single jobType. If the reload is tied to specific segment, then its a simple reload , otherwise reload_all. So, there's probably no real need to have 2 separate types. Also since we are creating Zk nodes per type, we wanted to restrict to 3 jobTypes. 
cc @saurabhd336